### PR TITLE
Quiz2

### PR DESCRIPTION
**Pull request by JasonWeakley**
Hokay so... I forgot to save changes to the JS file, so thats why I had to push twice in 5 minutes. This version of the branch has a working button and functions in the console! Yay!

### CODE REVIEW

**Comment by C-Stein** [comment]
Nice work! In a perfect world your argument and your variable name wouldn't be the same, but other than that, everything you have here looks good. The next step is to add 
-"Once the user enters in a number, and a character, the user can either then just press the enter key (as long as the cursor is in one of the input fields), or click a button that is labeled "Grow your tree" and the tree should be shown in the console." 
and 
-"If either of the input fields does not have a value in it when the user presses the enter key, or presses the button, then display an alert stating that both fields must have a value."
 Let me know when those are ready for grading and I'll check it out. As always, let us know if you have any questions.

**Comment by JasonWeakley** [comment]
Are you able to see the new changes as part of this pull request? Or do I need to cancel it and do a new one?

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	}
+}
+
+function isEmpty(textField){
+    if (textField == "") {
+    	alert("Please enter something you moron!");
+    }
+}
+
+function isNumber(textField) {
```
Comment (C-Stein) [inline quiz.js]: This alert shows up no matter what

**Comment by C-Stein** [comment]
Great work. The only thing is the alert that goes off no matter what, but that's a nice little extra you added that wasn't required by the directions. Go ahead an merge :+1:

**Comment by JasonWeakley** [comment]
Hmmm, thanks! I hadn't tested for that...

**Pull request merged**